src/components/VestingControl.tsx

```diff
@@ -48,11 +48,14 @@ const useStyles = makeStyles({
 export const VestingControl: React.FC<Props> = (props) => {
 	const context = useWeb3React();
 	const classes = useStyles();
-	const contracts = useContracts();
-	const vestingContract = contracts!.vesting;
+
+	//This lines must be changed so the user can set with ui 
+	//wich contract wants to send transactions, hardcoded actually to test
+	const contracts: Maybe<Contracts[]> = useContracts();
+	const vestingContract = contracts![0];
 	const { refreshToken, refresh } = useRefresh();
 	const vesting = useMerkleVesting(
-		vestingContract,
+		vestingContract!.vesting,
```

Comment:
`vestingContract.vesting` is a confusing name. `vestingContract` is actually a collection of contracts, and `.vesting` is an instance. The variable `vestingContract` should be renamed to something else, perhaps, `chosenContracts`